budgie-hotcorners/misc/meson.build

```diff
@@ -1,8 +1,25 @@
-install_data('defaults', install_dir: join_paths(datadir, 'budgie-hotcorners'))
-
 hotcorners_config_data = configuration_data()
 hotcorners_config_data.set('libexecdir', libexecdir)
 
+hotcorners_config_data.set('bdesktopdir', join_paths(datadir, 'budgie-desktop'))
+hotcorners_config_data.set('libdir', libdir)
+
+
+hotcorners_defaults = configure_file(
+    input: 'defaults.in.in',
+    output: 'defaults.in',
+    configuration: hotcorners_config_data,
+)
```

Comment:
Have we tried adding `install : true,
    install_dir : join_paths(datadir, 'budgie-hotcorners')` here?

Alternatively just use `install_data(hotcorners_defaults, install_dir: join_paths(datadir, 'budgie-hotcorners')`

The custom_target is primarily for updating a file with translations which we are not doing.

Reply:
We don't translate, but still need to replace the abs paths, hence the hotcorners_config_data?